src/cmdclass.py

```diff
@@ -31,7 +31,7 @@ def get_cmdclass(cmdclass=None):
     cmds = {} if cmdclass is None else cmdclass.copy()
 
     # we add "version" to both distutils and setuptools
-    from distutils.core import Command
+    from setuptools import Command
```

Comment:
IIUC setuptools monkey-patches `distutils.core`? I think to maintain current behavior while bypassing the deprecation:

```suggestion
    try:
        from setuptools import Command
    except ImportError:
        from distutils.core import Command
```